.github/workflow-templates/cargo-build/action.yml

```diff
@@ -64,14 +64,18 @@ runs:
       shell: bash
       run: |
         mkdir -p runtimes
-        cp target/release/wbuild/container-chain-template-simple-runtime/container_chain_template_simple_runtime.compact.wasm runtimes/;
-        cp target/release/wbuild/container-chain-template-frontier-runtime/container_chain_template_frontier_runtime.compact.wasm runtimes/;
+        cp target/release/wbuild/container-chain-template-simple-runtime/container_chain_template_simple_runtime.compact.compressed.wasm runtimes/
```

Comment:
are you intending to drop the semi-colon here?

Reply:
nope